src/Feliz.MaterialUI/Props.fs

```diff
@@ -1683,17 +1785,19 @@ type container =
   static member inline disableGutters (value: bool) = Interop.mkAttr "disableGutters" value
   /// Set the max-width to match the min-width of the current breakpoint. This is useful if you'd prefer to design for a fixed set of sizes instead of trying to accommodate a fully fluid viewport. It's fluid by default.
   static member inline fixed' (value: bool) = Interop.mkAttr "fixed" value
+  /// This component does not support children.
+  static member inline children  = UnsupportedProp ()
```

Comment:
It seems weird that a component called `container` may not have children (particularly since this was long supported). This may be a MUI documentation bug. Could you investigate? You may need to dig around in the MUI repo or raise an issue. If it is a MUI docs bug, we need to manually add in the generator.

Reply:
Seems to be related to this:  https://github.com/mui-org/material-ui/issues/21711

In short, the definition was changed to this: 
```
-    children: NonNullable<React.ReactNode>;
+    children?: React.ReactNode;
```

I added a comment to the issue. Not quite sure how to map the ```NonNullable<React.ReactNode>``` though.

Reply:
@cmeeren I tried adding code to add a children member for container at multiple places, but I didn't succeed. Neither adding it to ```additionalOverloads```, ```regularOverloads``` nor adding a "container" check to ```let addChildrenOverloadIfSupported (comp: Component) = ``` seems to make the children property of container anything other than ```UnsupportedProp ()```. Can you give me a hint where this would have to be added?

Reply:
`addChildrenOverloadIfSupported` adds a component constructor overload (i.e. to `Mui.componentName`).

I think you have to create a new function, say `addUndocumentedChildrenProp`, and chain it above `addChildrenOverloadIfSupported` where it's used. The function should, for now, simply add an appropriate `children` prop to the `container` component. If I have understood correctly, this prop should be the same as the old documented `children` prop (i.e., there should not be any changes to `Props.fs` regarding `container.children`.